### PR TITLE
Create polisth translate for checklist

### LINKED ISSUE #4: Polish translate
I've started creating Polish translate for this checklist :)

### PR DESCRIPTION
**Pull request by sdarmofal**
This is polish translate of checklist-en.md. It will be reviewed by my friends to make sure that this is correct.

Closes #4

### CODE REVIEW

File: checklist-pl.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## 1. Dozwolona składnia markdown na Devchecklists:
+  * [ ] Github usuwa białe znaki podczas kompilacji plików markdown. Aby utworzyć poprawny plik skorzystaj z [nieprzetworzonej wersji](https://raw.githubusercontent.com/vintasoftware/devchecklists-template/master/checklist-pl.md) tego pliku.
```
Comment (maarc91) [inline checklist-pl.md:2]: The link does not work for english version and it does not work for polish version. The Raw Button hiperlink is shorter

Compare with:
https://github.com/vintasoftware/devchecklists-template/blob/master/checklist-en.md

**Review by maarc91 — COMMENTED** [review]
Check the link to raw file

File: checklist-pl.md (hunk 1 above)
Comment (sdarmofal) [reply checklist-pl.md:2]: I think that the link should start working after merge. I specified link to this repository but file doesn't exist before merge.

Comment (maarc91) [reply checklist-pl.md:2]: Ok, got it :)

**Review by maarc91 — APPROVED** [review]

**Comment by sdarmofal** [comment]
@felipefarias @fjsj I think that it's ready to merge :)

**Pull request merged**